pages/linux/setfont.md

```diff
@@ -0,0 +1,20 @@
+# setfont
+
+> Set the terminal font.
```

Comment:
```suggestion
> Set the terminal font for TTYs.
```

Reply:
Isn't this kind of tautological. Almost the same as saying "Set the terminal font for terminals."

Reply:
It makes a distinction from terminal emulators and the tty IMO, someone may think this command will change their konsole or terminator for example.

Reply:
Ok, that's completely fair. I think it should be said in a way that newbies understand more easily, but I can't think of the words. We'll take this for now.